### PR TITLE
4 static code analysis

### LINKED ISSUE #4: Static code analysis
- SonarQube

### PR DESCRIPTION
**Pull request by csajkusz**
Closes #4

### CODE REVIEW

**Comment by csajkusz** [comment]
SonarCloud Code Analysis fails because code coverage is 0% because no unit test

File: .github/workflows/maven.yml [hunk 1]
```diff
@@ -5,20 +5,45 @@ name: Java CI with Maven
 
 on:
   push:
-    branches: [main]
+    branches:
+      - main
+      - 4-static-code-analysis
```
Comment (vabe) [inline .github/workflows/maven.yml]: Not necessary to include specific, one time branch here – we can remove it

**Review by vabe — APPROVED** [review]

**Comment by vabe** [comment]
> SonarCloud Code Analysis fails because code coverage is 0% because no unit test

Good, we can disable it from SonarCloud

**Comment by kajc10** [comment]
Conflicts resolved, ready to merge.

**Pull request merged**